### PR TITLE
AQ pubs :)

### PR DESCRIPTION
**Pull request by christopolise**
I am not certain if the way I referenced the journal is correct. But I thought it was worth a shot :D

### CODE REVIEW

File: _data/papers.yaml [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+  url: https://doi.org/10.3390/s23187721
+
 - title: Location Verification of Crowd-Sourced Sensors
   authors: C. Kitras, C. Pollan, K. Myers, C. Wirthlin Tischner, P. Lundrigan
   conference: The 2nd International Workshop on IoT in Emerging Fields (IoTEF), 2023
   abstract: Community-driven sensor networks have been instrumental in providing easy access to affordable, large-scale measurement recording, facilitated by the accessibility of inexpensive sensor hardware. The simplicity of this hardware makes it challenging to retrieve trustworthy location data without added hardware such as GPS. We introduce the LaMDA framework, a software-based solution run solely in a web browser to determine the location of a device with the aid of a registering device. We monitor the device for any location changes by analyzing its traceroute data from a central server. Our solution allows minimal firmware changes to be made to fleets of devices without recall or changes to hardware.
-  url: /assets/lambda.pdf
+  url: https://doi.org/10.1109/ICCCN58024.2023.10230111
```
Comment (philipbl) [inline _data/papers.yaml]: For papers behind a paywall, I like to host it myself so people can still get access to the paper. So I rather keep it how it was, unless you feel strongly about it.

Comment (christopolise) [reply _data/papers.yaml]: That's probably a valid point. I don't have too much of an opinion yet so that's okay

Comment (philipbl) [reply _data/papers.yaml]: Will you update the PR to remove this change?

Comment (christopolise) [reply _data/papers.yaml]: Finished

**Pull request merged**